src/OrchardCore.Modules/OrchardCore.OpenId/Migrations/DefaultScopesMigration.cs

```diff
@@ -19,18 +19,6 @@ public int Create()
             var S = shellScope.ServiceProvider.GetService<IStringLocalizer<DefaultScopesMigration>>();
             var scopeManager = shellScope.ServiceProvider.GetRequiredService<IOpenIdScopeManager>();
 
-            if (await scopeManager.FindByNameAsync(OpenIddictConstants.Scopes.Profile) == null)
```

Comment:
Out of curiosity, why this change?

Reply:
Because `openid` and `profile` scopes are always included with every request. So I don't think we need the user to explicitly including these 2.

Reply:
Not all OIDC clients request the `profile` scope. And in a (distant) future, we'll want to update the consent view to display the scope description (and potentially allow users to deselect some of the requested scopes).

I'd keep it.

Reply:
done.